app/src/components/Home.js

```diff
@@ -1,32 +1,27 @@
-import React, { Component } from 'react'
+import React, { useState, useEffect } from 'react'
 import './Home.sass'
 import Carousel from './Carousel'
 import { retrieveUser } from 'client-logic'
 
-export default class extends Component {
-  constructor () {
-    super()
+const Home = () => {
+  const [user, setUser] = useState()
```

Comment:
I would change the name of the state to username because your are not saving an user object, you only save the name